iotcore/src/test/java/com/google/android/things/iotcore/ConnectionParamsTest.java

```diff
@@ -245,4 +245,12 @@ public void testGetConfigurationTopic() {
         String expectedConfigurationTopic = "/devices/" + DEVICE_ID + "/config";
         assertThat(connectionParams.getConfigurationTopic()).isEqualTo(expectedConfigurationTopic);
     }
+
+    @Test
+    public void testGetCommandsTopicPrefix() {
+        ConnectionParams connectionParams = buildAllParameters();
+
+        String expectedCommandsTopicPrefix = "/devices/" + DEVICE_ID + "/commands";
```

Comment:
the variable name would be "expectedCommandsTopic"

Reply:
my bad: sloppy...